OneSignalSDK/onesignal/src/main/java/com/onesignal/OSUtils.java

```diff
@@ -667,4 +669,19 @@ static Throwable getRootCauseThrowable(@NonNull Throwable subjectThrowable) {
    static String getRootCauseMessage(@NonNull Throwable throwable) {
       return getRootCauseThrowable(throwable).getMessage();
    }
+
+   /**
+    * Provides a way to check if WorkManager is initialized in this process.
+    *
+    * This is effectively the WorkManager.isInitialized() public method introduced in androidx.work:work-*:2.8.0-alpha02.
+    * Please see https://android-review.googlesource.com/c/platform/frameworks/support/+/1941186.
+    *
+    * @return {@code true} if WorkManager has been initialized in this process.
+    */
+   @SuppressWarnings("deprecation")
+   @SuppressLint("RestrictedApi")
+   static boolean isWorkManagerInitialized() {
```

Comment:
Would be cleaner to put this into a new WorkManagerHelper class instead of adding to this OSUtil misc class.

Reply:
Done